code/controllers/mc/master.dm

```diff
@@ -124,15 +124,21 @@ var/CURRENT_TICKLIMIT = TICK_LIMIT_RUNNING
 // 	Make a subsystem, give it the SS_NO_FIRE flag, and do your work in it's Initialize()
 /datum/controller/master/proc/Setup()
 	set waitfor = FALSE
-	sleep(1 SECONDS)
-	to_chat(world, "<span class='boldannounce'>Initializing subsystems...</span>")
-
+	sleep(-1)
+	//moving this random bullshit into here, because it didn't belong in world/New()
+	//SetupHooks() // /N3X15 project from 8 years ago (WIP)
```

Comment:
A quick search indicates there are 7 `/hook_handler`s in use and 18 `/hook`s defined, yet you decided to disable them. Why?

Reply:
N3X15 said he didn't test it and didn't get it working but I did miss that... I can undo it